core/src/watchtower.rs

```diff
@@ -1,7 +1,152 @@
-use crate::{database::Database, extended_rpc::ExtendedRpc};
+use crate::{
+    actor::{Actor, WinternitzDerivationPath},
+    config::BridgeConfig,
+    database::Database,
+    errors::BridgeError,
+    extended_rpc::ExtendedRpc,
+    rpc::{
+        self,
+        clementine::{
+            clementine_operator_client::ClementineOperatorClient,
+            clementine_verifier_client::ClementineVerifierClient,
+        },
+    },
+};
+use bitvm::signatures::winternitz;
 
 #[derive(Debug, Clone)]
 pub struct Watchtower {
-    _rpc: ExtendedRpc,
+    _erpc: ExtendedRpc,
     _db: Database,
+    pub actor: Actor,
+    num_operators: u32,
+    num_time_tx: u32,
+    pub index: u32,
+    pub(crate) _verifier_clients: Vec<ClementineVerifierClient<tonic::transport::Channel>>,
+    pub(crate) _operator_clients: Vec<ClementineOperatorClient<tonic::transport::Channel>>,
+}
+
+impl Watchtower {
+    pub async fn new(config: BridgeConfig) -> Result<Self, BridgeError> {
+        let _erpc = ExtendedRpc::new(
+            config.bitcoin_rpc_url.clone(),
+            config.bitcoin_rpc_user.clone(),
+            config.bitcoin_rpc_password.clone(),
+        )
+        .await;
+        let _db = Database::new(&config).await?;
+        let actor = Actor::new(
+            config.secret_key,
+            config.winternitz_secret_key,
+            config.network,
+        );
+
+        let verifier_endpoints =
```

Comment:
Watchtower server does not need verifier endpoints and operator endpoints

Reply:
Fixed in https://github.com/chainwayxyz/clementine/commit/1f907464d6118da689cdc1e91759010445a713aa